src/py/flwr/server/superlink/driver/driver_servicer.py

```diff
@@ -174,14 +175,22 @@ def GetRun(
                 fab_id=run.fab_id,
                 fab_version=run.fab_version,
                 override_config=user_config_to_proto(run.override_config),
+                fab_hash=run.fab_hash,
             )
         )
 
     def GetFab(
         self, request: GetFabRequest, context: grpc.ServicerContext
     ) -> GetFabResponse:
-        """Will be implemented later."""
-        raise NotImplementedError
+        """Get FAB from Ffs."""
+        log(DEBUG, "DriverServicer.GetFab")
+
+        ffs: Ffs = self.ffs_factory.ffs()
+        if result := ffs.get(request.hash_str):
+            fab = Fab(request.hash_str, result[0])
+            return GetFabResponse(fab=fab_to_proto(fab))
+
+        return GetFabResponse()
```

Comment:
We should probably use a gRPC error code here

Reply:
@danieljanes I tried to replicate what we do in other functions (like `GetRun`). Should I open a separate PR to also modify `GetRun`'s behaviour?